Update Shadow DOM topic (#6295)
DevExpress/devextreme-documentation: concepts/Common/Shadow DOM/10 Benefits.md

> Shadow DOM offers the following benefits for web development:

Reviewer: ```suggestion
The Shadow DOM offers the following benefits for web development:
```

- **Encapsulation**    

> Shadow DOM isolates component's styles and scripts. This isolation prevents conflicts between the component and other styles and scripts on the page.

Reviewer: ```suggestion
The Shadow DOM isolates a component's styles and scripts. This isolation prevents conflicts between the component and other styles and scripts on the page.
```

> Shadow DOM isolates component's styles and scripts. This isolation prevents conflicts between the component and the other styles and scripts on the page.

Reviewer 2: ```suggestion
Shadow DOM isolates component's styles and scripts. This isolation prevents conflicts between the component and other styles and scripts on the page.
```

- **Reusability**     
Encapsulation makes Shadow DOM components self-contained, and you can use them across various parts of an application or even different applications.

- **Subtree Independence**    

> Shadow DOM creates isolated environments for components. This functionality allows independent development, testing, and implementation, which improves maintainability of large applications.

Reviewer: ```suggestion
The Shadow DOM creates isolated environments for components. This functionality allows independent development, testing, and implementation, which improves maintainability of large applications.
```

- **Performance Improvement**    

> Shadow DOM encapsulates styles and code, allowing browsers to update only specific page parts, which enhances performance.

Reviewer 2: ```suggestion
Shadow DOM encapsulates styles and code. As a result, it allows browsers to update only specific page parts and enhances performance.
```

> Shadow DOM encapsulates styles and code. As a result, it allows browsers to update only specific page parts and enhances performance.

Reviewer: ```suggestion
The Shadow DOM encapsulates styles and code. As a result, it allows browsers to update only specific page parts and enhances performance.
```
